environment.yml

```
@@ -5,6 +5,6 @@ channels:
 dependencies:
   - python=3.9
   - gmt=6.2.0
-  - pygmt=0.3.1
+  - pygmt=0.4.0
   - jupyterlab=3.0.7
```

Just a little version bump to trigger a new build. Will commit once PyGMT v0.4.0 lands on https://anaconda.org/conda-forge/pygmt/files.

```suggestion
  - jupyterlab=3.0.16
```